### ISSUE TITLE
[Menu redesign] [Accessibility] - "Tools" and "Save" headers are not announced

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#23050]
**Steps to reproduce**

1. Install FF.
1. Enable voice over.
1. Open FF and access a webpage.
1. Tap on the hamburger menu.
1. Go to Tools submenu and listen. Access the submenu and listen  the header announcement.
1. Go to Save submenu and listen. Access the submenu and llisten  the header annuncement.

**Expected behavior**
”Tools” and “save” menu names are  announced.

**Actual behavior**
”Tools” and “save” menu names are not announced.

**Device & build information**

* Device: iPhone 16+ (18.1)
* Build version: v133 (47401)
* First seen version: ?

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-10512)

**Comment by anna-yeddi** [comment]
This issue does not seem to be a blocker of the experience, but would definitely make it harder to understand and use for a users who cannot see the visual hierarchy and/or who relies on a screen reader to understand it.

**Comment by data-sync-user** [comment]
➤ Diana Andreea Barladeanu commented:

Verified as fixed on v133 (47631), with iPhone 15 (17.6).
